src/exercise2.py

```diff
@@ -0,0 +1,16 @@
+import re
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route('/password/<string:password>', methods = ['GET'])
+def validate(password):
```

Comment:
The password should not be passed via the url parameters.
It can be passed in the api body